charts/identity-server/values.prod.yaml

```diff
@@ -201,6 +201,7 @@ services-auth-admin-api:
     DB_REPLICAS_HOST: 'postgres-ids.internal'
     DB_USER: 'servicesauth'
     IDENTITY_SERVER_ISSUER_URL: 'https://innskra.island.is'
+    IDENTITY_SERVER_ISSUER_URL_LIST: '"https://innskra.island.is"'
```

Comment:
Yes. My thinking is that we are adding opt-in multi issuer of JWT tokens for API authentication. But for auth-admin-api in production it will only allow production tokens, so using single issuer. [This new function](https://github.com/island-is/island.is/pull/10387/files#diff-f5f9c1932e53e1e81509e25f5341b2942d089f6a37855b6e98fdda530ce11aa1R17) introduced in this PR is always checking if issuer is provided as an `string` or `string[]`  on app startup. But the configuration for the app is just the same and is not bloated by some array checking. 

I could do a additional smarter thing, if `createKeyProvider` is provided with array of single item it can just use the default method. So then this doesn't matter if it is string or array for single item.

Reply:
ok, I dont mean to be annoying but `IDENTITY_SERVER_ISSUER_URL_LIST` looks confusing when given a string